### ISSUE TITLE
[BUG] MCDU should reset .

### ISSUE THREAD

**Issue by mikecbr** [flybywiresim/a32nx#2193]
**Mod Version**
Latest branch

**Describe the bug**
The MCDU should reset almost all the parametres once you( turn off the engines at the end of a flight )  and should reset everything in the MCDU   , it should  appear like when is the first time you turn on the aircraft in the fs2020  in the stand. (Cold and dark).
Not really sure now  if it is once you just land , or what i said before , when you turn off the engines

**To Reproduce**

1.Make a flight ,turn off completely 
2.Turn it on again and the MCDU shows you the data from the last flight you did departure/ destination
3.

**Expected behavior**

 it should  appear like when is the first time you turn on the aircraft in the fs2020  in the stand. (Cold and dark).

**Actual behavior**
All the parametres in the MCDU still there and also you can not clear most of them 

**References**

**Additional context**
Also you can not clear some data in the MCDU

Was this working before/when did the issue start occurring?
No

Is this a problem in the vanilla unmodded game?
No

Discord username (if different from GitHub):

**Comment by beheh** [comment]
To expand on this issue, I believe it resets already as part of the FMGC phase "DONE" which should be triggered 30 seconds after landing. At that point the MCDUs revert to the DATA page and, as mentioned, the FMGC resets most values.

**Comment by Hundredabove** [comment]
@beheh  yep  exacly thanks for the explanation

**Comment by Benjozork** [comment]
Duplicate of #288
